src/ILCompiler.MetadataTransform/tests/PrimaryMetadataAssembly/Platform.cs

```diff
@@ -56,7 +56,7 @@ public class Array<T> : Array, System.Collections.Generic.IList<T> { }
 
     public class Exception { }
 
-    public class __ComObject : Private.CompilerServices.ICastable { }
+    public class __ComObject : Private.CompilerServices.IBlockedInterface { }
```

Comment:
It would be nice to rename `__ComObject` to something else in the MetadataTransform test, e.g. `BlockedObject`.